### PR TITLE
Add tests

### PR DESCRIPTION
**Pull request by robinandeer**
Tests and fixes

### CODE REVIEW

File: puzzle/plugins/vcf.py [hunk 1]
```diff
@@ -248,9 +248,10 @@ def variant(self, case_id, variant_id):
             Returns:
                 variant (Variant): The variant object for the given id
         """
-        for variant in self.variants(case_id):
-            if variant['variant_id'] == variant_id:
-                return variant
+        for variant_obj in self.variants(case_id, count=float('inf')):
```
Comment (robinandeer) [inline puzzle/plugins/vcf.py]: This fixes the issue @dnil mention about variants not loading after "load more" (hahaha)

The skip/limit stuff was in the way :P

Comment (moonso) [inline puzzle/plugins/vcf.py]: Perfect!

File: puzzle/plugins/vcf.py [hunk 2]
```diff
@@ -248,9 +248,10 @@ def variant(self, case_id, variant_id):
...
+            print(variant_obj['variant_id'])
```
Comment (moonso) [inline puzzle/plugins/vcf.py]: Why ```print```?

Comment (robinandeer) [inline puzzle/plugins/vcf.py]: just something I used to debug :stuck_out_tongue_closed_eyes:

**Pull request merged**